observability/network_observability/network-observability-secondary-networks.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="network-observability-secondary-networks"]
+= Secondary networks
+include::_attributes/common-attributes.adoc[]
+:context: secondary_networks
+
+toc::[]
+You can configure the Network Observability Operator to collect and enrich network flow data from secondary networks, such as SR-IOV and Open Virtual Network (OVN)-Kubernetes.
+
+// Note to tech review:
+// Is the existing SR-IOV example we have, "Configuring monitoring for SR-IOV interface traffic", an example of secondary network? If so, it is not through a VM, right? 
+
+.Prerequisites
+* Access to an {product-title} cluster with an additional network interface, such as a secondary interface or an L2 network.
+
+include::modules/network-observability-SRIOV-configuration.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+For more information about creating the `SriovNetwork` custom resource, see xref:../../networking/hardware_networks/configuring-sriov-device.adoc#cnf-creating-an-additional-sriov-network-with-vrf-plug-in_configuring-sriov-device[Creating an additional SR-IOV network attachment with the CNI VRF plugin].
```

Comment:
Should be a bulleted list, and normally should just be a link and not a sentence

```suggestion
* xref:../../networking/hardware_networks/configuring-sriov-device.adoc#cnf-creating-an-additional-sriov-network-with-vrf-plug-in_configuring-sriov-device[Creating an additional SR-IOV network attachment with the CNI VRF plugin]
```

Reply:
Right. I copied this from an assembly that pre-dated the decision to just use bullets, and this needed to be updated. Thanks for pointing it out.